app/factories/LobbyFactory.js

```diff
@@ -8,11 +9,24 @@ function LobbyFactory(){
     var lobby;
 
     lobby = {
-      name: options.name || "",
-      hostSettings: options.hostSettings || [],
-      playerSettings: options.playerSettings || []
+      //Just for test we disable this
+      //name: options.name || "",
+      //hostSettings: options.hostSettings || [],
```

Comment:
I know how they work but I commented them because before I start working on them I enabled a options which already are configured XD @MythicManiac Also they didnt work and I'll solve later